README.md

```diff
@@ -1,2 +1,13 @@
 # TSDemoBoard
 Jupyter notebooks for showing the evolving set of services configured and implemented in a HAPI-FHIR server.
+
+## Setup
+1. Clone repository
+2. Make a virtual environment (optional)
+3. Run: `pip install -r requirements.txt`
+
+## Running
+1. From command line, change directory to where you cloned this repository.
+2. Run: `jupyter notebook`
```

Comment:
**Documentation**
@chrisroederucdenver I added info to the `README.md` regarding how to set up and run. Not the best instructions ever, but good enough for now. Feel free to tweak.

Also, because these instructions are here now, I deleted `start-jupyter.sh`. But I can undo that and add it back if you want.